docs/tests/tests_modules.rst

```diff
@@ -3,7 +3,7 @@ Test Scripts
 
 .. toctree::
 
-    test_ai_helpers
+    test_q_learning
```

Comment:
Need to add DQN to documentation. Check Env will be used as a test and nothing more.